data/DC/incentives.json

```diff
@@ -0,0 +1,330 @@
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-15",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "electric_stove",
+    "program": "dc_dCSustainableEnergyUtility_affordableHomeElectrificationProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-16",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "dc_dCSustainableEnergyUtility_affordableHomeElectrificationProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 6000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-19",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "electric_panel",
+    "program": "dc_dCSustainableEnergyUtility_affordableHomeElectrificationProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2300
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-21",
+    "authority_type": "state",
+    "authority": "dc-dc-department-of-energy-and-environment",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "dc_dCDepartmentOfEnergyAndEnvironment_weatherizationAssistanceProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Provides low-income residents with technical and financial assistance to help reduce their energy bills by making their homes more energy efficient."
```

Comment:
Also not like most of our descriptions – lacks specificity around what work is done and what the financial arrangement is.